src/streamlink/plugins/afreeca.py

```diff
@@ -50,7 +50,7 @@ class AfreecaHLSStream(HLSStream):
     help="Purge cached AfreecaTV credentials to initiate a new session and reauthenticate.",
 )
 class AfreecaTV(Plugin):
-    _re_bno = re.compile(r"var nBroadNo = (?P<bno>\d+);")
+    _re_bno = re.compile(r"window.nBroadNo = (?P<bno>\d+);")
```

Comment:
```suggestion
    _re_bno = re.compile(r"window\.nBroadNo\s*=\s*(?P<bno>\d+);")
```

The dot needs to be escaped